pkg/tasks/controlplane.go

```diff
@@ -77,45 +76,20 @@ func (t *ControlPlaneTask) Run(ctx context.Context) error {
 		}
 	}
 
-	sms, err = t.factory.ControlPlaneEtcdServiceMonitors()
+	// NOTE: This is temporary, to clean these resources that used to be managed by CMO, now
```

Comment:
I'll create a task for this and assign it to me.

Reply:
Maybe one also to move RBAC related resources as well https://github.com/openshift/cluster-monitoring-operator/blob/b886eb231ddf370b577ec4e572b1563d8889e28f/assets/prometheus-k8s/role-binding-specific-namespaces.yaml#L60-L78 (if possible)

Reply:
Does the etcd operator create this `RoleBinding` already? Our code does a noop if the RB already exists, so it should be safe to create them on either/both sides. Not really sure what the best practice is. CMO only creates RBs for SMs that it sets up.
So sounds like CEO would be the right place.

Reply:
No, it doesn't create the `Role/RoleBinding` for the `openshift-etcd` namespace.
Yes, it makes more sense for me too to delegate this to CEO, I'll create the ticket then, maybe aim for 4.14 as well.

Reply:
sure, let's move it to CEO as soon as convenient.